roles/openshift_on_openstack_secgroup/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# The path to the OpenStack RC file on the openstack-server, may be named differently than other servers.
+openstack_rc: "{{ lookup('env', 'openstack_rc_path')|default(ansible_user_dir ~ '/overcloudrc', true) }}"
+# OpenStack security group name to add inbound rules to.
+security_group_name: 'openshift-ansible-scale-ci.example.com-common-secgrp'
+# TCP/UDP inbound port range for the uperf client.
+#uperf_range: '20000:20200'
+uperf_range: '1024:65535'
```

Comment:
There has to be a better way than disabling all the ports! 

After discussing with @jmencak we agreed that trying to get all the specific ports could take forever. 

I would like to document a way to undo this change in case we want to test as customers would with the default security groups.